tests/php/tests/includes/helper/test_class.wp-job-manager-helper-api.php

```diff
@@ -225,6 +225,24 @@ protected function default_invalid_response() {
 	}
 
 	protected function build_url( $args ) {
+		// The legacy endpoints are temporary. For now, translate `license_key` => `licence_key` at this point.
+		$args = $this->updateArrayKey( $args, 'license_key', 'licence_key' );
+
 		return 'https://wpjobmanager.com/?' . http_build_query( $args, '', '&' );
 	}
+
+	private function updateArrayKey( $array, $oldKey, $newKey ) {
+		if ( array_key_exists( $oldKey, $array ) ) {
```

Comment:
This method is used to keep the arguments in the same order to match the transport faker hash (See `Requests_Transport_Faker::make_request_signature`).